pages/linux/iw.md

```diff
@@ -0,0 +1,19 @@
+# iw
+
+> Show / manipulate wireless devices.
```

Comment:
Perhaps `Show and manipulate`... would read better?

Reply:
I agree.
However, I choose this sentence form because it is already used for [`ip` tldr page](https://github.com/tldr-pages/tldr/blob/master/pages/linux/ip.md).